src/swagger-specs/streaming-ingestion.yml

```diff
@@ -0,0 +1,638 @@
+          imsOrgId: '5C1328435BF324E90A49402A@AdobeOrg'
+          datasetId: '5da9452f7de80400007jc52a'
+          createdAt: "1526283801869"
+          flowId: "69f188b6-af07-4587-8061-e94d34637b41"
+          source:
+            name: 'Google BigQuery Base Connection'
+        body:
+          xdmMeta:
+            schemaRef:
+              id: https://ns.adobe.com/{TENANT_ID}/schemas/{SCHEMA_ID}
+              contentType: application/vnd.adobe.xed-full+json;version={SCHEMA_VERSION}
+          xdmEntity:
+            person:
+              name:
+                firstName: Jane
+                middleName: F
+                lastName: Doe
+              birthDate: 1969-03-14
+              gender: female
+            workEmail:
+              primary: true
+              address: janedoe@example.com
+              type: work
+              status: active
+    StreamingIngestionResponse:
+      type: object
+      properties:
+        inletID:
+          type: string
+          description: The ID of the previously created streaming endpoint.
+        xactionId:
+          type: string
+          description: The `xactionID` is a unique, server-side generated identifier 
+            for the XDM record that you just sent. This ID helps with debugging and 
+            for Adobe to trace this record's lifecycle through it's various systems.
+        receivedTimeMs:
+          type: integer
+          description: The timestamp in **milliseconds** that shows the time at which
+            the request was received.
+          format: int32
+      example:
+        inletId: '667b41cf2dbf3509927da1ebf7e93c20afa727cc8d8373e51da18b62e1b985ec'
+        xactionId: 1532625558467:0001:13
+        receivedTimeMs: "1551307525735"
+    BatchMessage:      
+      type: object
+      description: The batch message to be published to Platform.
+      properties:
+        messages:
+          $ref: '#/components/schemas/BatchMessageArray'
+    messageHeader:
+      required:
+        - schemaRef
+        - imsOrgId
+        - datasetId
+      type: object
+      description: This payload header describes the payload information and is 
+        necessary for the transport and reconstruction of the original resource. 
+        It ensures that the packet is received at it's destination.
+      properties: 
+        schemaRef:
+          $ref: '#/components/schemas/SchemaReference'
+        imsOrgId:
+          type: string
+          description: Your organization ID. It can be found under the integration details in the Adobe Developer Console.
+        datasetId:
+          type: string
+          description: The ID of the dataset that you want to publish to.
+        createdAt: 
+          type: string
+          description: A timestamp to date the creation of the connection. This is automatically generated.
+        flowId: 
+          type: string
+          description: The dataflow ID. This is useful when there are 
```

Comment:
I think this can just be the "The ID of the dataflow." IF you want to elaborate further, maybe something like "Use the flow ID to query against a specific dataflow, when the account you are using has been used to create multiple dataflows."